pkg/devfile/adapters/kubernetes/component/status_test.go

```diff
@@ -162,12 +164,30 @@ func TestGetDeploymentStatus(t *testing.T) {
 
 			comp := testingutil.GetFakeContainerComponent(testComponentName)
 			devObj := devfileParser.DevfileObj{
-				Data: &testingutil.TestDevfileData{
-					Components: []devfilev1.Component{comp},
-					Commands: []devfilev1.Command{
-						getExecCommand("run", devfilev1.RunCommandGroupKind),
-					},
-				},
+				Data: func() data.DevfileData {
+					devfileData, err := data.NewDevfileData(string(data.APIVersion200))
+					if err != nil {
+						t.Error(err)
+					}
+					if err != nil {
+						t.Error(err)
+					}
+					err = devfileData.AddComponents([]devfilev1.Component{comp})
+					if err != nil {
+						t.Error(err)
+					}
+					if err != nil {
+						t.Error(err)
+					}
+					err = devfileData.AddCommands([]devfilev1.Command{getExecCommand("run", devfilev1.RunCommandGroupKind)})
+					if err != nil {
+						t.Error(err)
+					}
+					if err != nil {
+						t.Error(err)
+					}
```

Comment:
And at a bunch of places above that as well. Fixed.

Reply:
That happened because I used a few `sed` one-liners to add error check at a bunch of places to avoid having to do it all manually:
```sh
#!/bin/sh

sed -i 's/devfileData, _ \:\=/devfileData, err \:\=/g'  $1
sed -i '/devfileData, err \:\=/a if err!=nil \{\n t.Error(err) \n \}' $1
sed -i 's/_ \= devfileData\.Add/err = devfileData\.Add/g' $1
sed -i '/err \= devfileData.Add/a if err!=nil \{\n t.Error(err) \n \}' $1

go fmt $1
go test -i -race $1
echo $?
```